src/app/authorization/login/login.component.ts

```diff
@@ -60,17 +68,41 @@ export class LoginComponent implements OnInit, OnDestroy {
       let data: any = localStorage.getItem('registeruser');
       this.userData = JSON.parse(data);
     }
-    this.createForm();
+  }
+
+  signout() {
+    localStorage.removeItem('isAuthenticate');
+    document.cookie = 'username' + '=' + null;
+    this.route.navigateByUrl('/login');
+  }
+
+  initForm() {
```

Comment:
Why there is two function that is creating form first one is initForm and createForm() please remove this

Reply:
removed initForm() function and done all functionality in createForm() function

Reply:
okay . i removed initForm function and make all the functionality in the createForm function.